src/Context/actions.js

```diff
@@ -8,33 +10,60 @@ export const getTeams = () => async (dispatch) => {
   };
 
   const response = await fetch(
-    encodeURI('https://api.pagerduty.com/teams'),
+    encodeURI('https://api.pagerduty.com/users/me'),
```

Comment:
This request should be wrapped in try/catch, similar to how `getTeams()` handles it.